### DISCUSSION TITLE
The bug template of this repo is flawed!

### DISCUSSION THREAD

**Discussion by mike-lischke** [eslint/rewrite#98] [General]
I'm currently trying to report a bug of the rewrite tool, but am unable to finish, because it is missing something without telling what. The "Submit new issue" button simply stays disabled. I assume it has to do with the fact that I cannot provide a stackblitz example, but hey, not every problem can be demonstrated by such an example. The template should cater for that! Also, it seems as if it was copied from ESLint without adjustment, so it says "Report a problem with ESLint." (instead of the rewrite tool) and lists a number of affected packages, which I'm not sure they are related to the config rewriting process.

Anyway, I want to report that the config migrations fails on macOS with a node version installed by homebrew. How can I provide a reproducible example on Stackblitz for that? 

The problem is that the migration tool tries to access a file that does not exist:

> npx @eslint/migrate-config .eslintrc.json --gitignore
> npm ERR! code ENOENT
> npm ERR! syscall lstat
> npm ERR! path /opt/homebrew/Cellar/node/22.5.1/lib
> npm ERR! errno -2
> npm ERR! enoent ENOENT: no such file or directory, lstat '/opt/homebrew/Cellar/node/22.5.1/lib'
> npm ERR! enoent This is related to npm not being able to find a file.
> npm ERR! enoent 

This node installation works fine, so apparently there's simply no /lib file/folder in this version.

**Comment by mike-lischke** [comment]
Turns out Node.js installed by homebrew is flawed. Using `nvm` to install it went much better and the rewrite tool runs now properly. So the error mentioned above is solved. However, the original issue still exists. It's impossible to file a bug report in such a situation.

**Comment by nzakas** [comment]
Thanks for the feedback. As you've noticed, the problem wasn't with the migrate tool but rather with how you had everything installed. By asking for a repro case, it allows us to filter out problems such as this, which are not bugs.

You can always open a Discussion if you just need help and aren't reporting a bug.
